client/package.json

```diff
@@ -100,7 +101,11 @@
     "tus-js-client": "^3.1.1",
     "underscore": "^1.13.6",
     "util": "^0.12.5",
+    "vega": "^5.30.0",
+    "vega-embed": "^6.26.0",
+    "vega-lite": "^5.21.0",
```

Comment:
Since this is only needed in a few places, and quite a large package, could we load this on demand?

Reply:
I suppose if you ask then it is possible 😃 ? I don't know how to do that though.

Reply:
I can push a commit if you like

Reply:
To explain what's happening here, to anyone curious:

- the vega libs were added to "modulesExcludedFromLibs" so it's not bundled into the libs bundle. This adds these libs to their own bundles.
- type imports were changes from `import { type ...}` to `import type { ... }`. These are not equivalent. The second one let's typescript completely remove the import line, while the first keeps an empty import, in case of import side effects. This would break on demand loading
- Finally the component is imported as `const VegaWrapper = () => import("./VegaWrapper.vue");`. This is a way to lazy load vue components and all it's related code (the vega library in this case). The component can be used like before